core/controllers/learner_playlist.py

```diff
@@ -27,12 +27,31 @@ class LearnerPlaylistHandler(base.BaseHandler):
     @acl_decorators.can_access_learner_dashboard
     def post(self, activity_type, activity_id):
         position_to_be_inserted_in = self.payload.get('index')
+
+        playlist_limit_exceeded = False
+        belongs_to_completed_or_incomplete_list = False
```

Comment:
No we don't prirotitize the messages. I have changed the code accordingly. All the information text are important. If the user fails to add an exploration to the playlist he/she should know all the information. Or else if the user tries to add an exploration that he/she has created, but is not added because 1) it is created by him/her. 2) the playlist limit has exceeded. If we only show the first message the user will try to add another exploration instead of completing activities in the playlist.